powerscale/provider/synciqpolicy_resource_schema.go

```diff
@@ -0,0 +1,635 @@
+				Validators: []validator.String{
+					stringvalidator.LengthBetween(1, 4096),
+				},
+			},
+			"target_snapshot_alias": schema.StringAttribute{
+				Optional:            true,
+				Computed:            true,
+				Description:         "The alias of the snapshot taken on the target cluster after the sync completes. A value of @DEFAULT will reset this field to the default creation value.",
+				MarkdownDescription: "The alias of the snapshot taken on the target cluster after the sync completes. A value of @DEFAULT will reset this field to the default creation value.",
+				Validators: []validator.String{
+					stringvalidator.LengthBetween(0, 255),
+				},
+			},
+			"target_snapshot_archive": schema.BoolAttribute{
+				Optional:            true,
+				Computed:            true,
+				Description:         "If true, archival snapshots of the target data will be taken on the target cluster after successful sync completions.",
+				MarkdownDescription: "If true, archival snapshots of the target data will be taken on the target cluster after successful sync completions.",
+			},
+			"target_snapshot_expiration": schema.Int64Attribute{
+				Optional:            true,
+				Computed:            true,
+				Description:         "The length of time in seconds to keep snapshots on the target cluster.",
+				MarkdownDescription: "The length of time in seconds to keep snapshots on the target cluster.",
+			},
+			"target_snapshot_pattern": schema.StringAttribute{
+				Optional:            true,
+				Computed:            true,
+				Description:         "The name pattern for snapshots taken on the target cluster after the sync completes.  A value of @DEFAULT will reset this field to the default creation value.",
+				MarkdownDescription: "The name pattern for snapshots taken on the target cluster after the sync completes.  A value of @DEFAULT will reset this field to the default creation value.",
+			},
+			"workers_per_node": schema.Int64Attribute{
+				Optional:            true,
+				Computed:            true,
+				Description:         "The number of worker threads on a node performing a sync.",
+				MarkdownDescription: "The number of worker threads on a node performing a sync.",
+			},
+			"conflicted": schema.BoolAttribute{
+				Optional:            true,
+				Computed:            true,
+				Description:         "NOTE: This field should not be changed without the help of PowerScale support.  If true, the most recent run of this policy encountered an error and this policy will not start any more scheduled jobs until this field is manually set back to 'false'.",
+				MarkdownDescription: "NOTE: This field should not be changed without the help of PowerScale support.  If true, the most recent run of this policy encountered an error and this policy will not start any more scheduled jobs until this field is manually set back to 'false'.",
+			},
+			"database_mirrored": schema.BoolAttribute{
```

Comment:
Computed attributes are needed in resource? If not, can we use data source for the same?

Reply:
done